packages/application-shell/src/feature-toggles.js

```diff
@@ -11,4 +11,3 @@
  */
 
 // eslint-disable-next-line import/prefer-default-export
-export const ORGANIZATIONS_LIST = 'organizationsList';
```

Comment:
This file does not seem to be used, can we remove it?

Reply:
I'd say lets keep it in case we add feature toggles. My intend to keep it was to keep git history a bit easier to follow with when toggles got removed or added.

Reply:
Sure but even in this case it wasn't being used, as the org list toggle was injected directly.
Anyway, it's fine either way.

Reply:
Yeah cause in quick access they're all for some strange reason.